### DISCUSSION TITLE
Tool & Work frames

### DISCUSSION THREAD

**Discussion by stephenfelix** [Yaskawa-Global/YMConnect#8] [Q&A] [answered]
Does YMConnect as of now setting the tool definitions and and coordinate systems?
Similarly motion influenced by these definition such as `mpMOVL` is the MotoPlus API.

**Comment by ted-miller** [comment, marked as answer]
Hello @stephenfelix 
Yes. When you submit a motion target, you specify a `PositionData` structure. Within this structure, the tool-number and coordinate-system are specified. https://developer.motoman.com/en/YMConnect/DataStructures#positiondata

Please note that the MotoPlus API `mpMOVL` is used for point-to-point motion only. The robot must fully decelerate between each point.

The motion interface in YMConnect is the same as the `mpMot___` API's in MotoPlus. This allows for a smooth continuous trajectory through a series of waypoints.
